spec/models/like_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'tests that Like model is created correctly' do
+    like = Like.new
+    like.build_user(name: 'Somdotta')
+    like.build_post(text: '')
+    expect(like).to be_valid
+  end
+
+  it 'updates a posts likes correctly' do
+    user = User.new(name: 'Tom', posts_counter: 0)
+    user2 = User.create!(name: 'Lilly', posts_counter: 0)
+    post = user2.posts.create!(title: 'Post1', text: 'This is a post', likes_counter: 0, comments_counter: 0)
+    post.likes.create!(user:)
```

Comment:
- [ ] On line no. 15, I think you have missed mentioning user in the parameter. It should be like:

`post.likes.create!(user: user)`.

Reply:
Thanks for your time and feedback. I'll fix it ASAP.